src/structs/residue.rs

```diff
@@ -339,3 +339,5 @@ impl PartialEq for Residue {
             && self.atoms == other.atoms
     }
 }
+
+impl Eq for Residue {}
```

Comment:
As Residue contains multiple f64 values it cannot implement Eq as f64 does not implement Eq.